zulipterminal/ui.py

```diff
@@ -133,17 +134,17 @@ def main_window(self) -> Any:
         self.left_panel = self.left_column_view()
         self.center_panel = self.middle_column_view()
         self.right_panel = self.right_column_view()
-        if self.controller.autohide:
+        if self.autohide:
             body = [
-                (View.LEFT_WIDTH, self.left_panel),
+                (self.LEFT_WIDTH, self.left_panel),
```

Comment:
There's a hybrid in the rest of the PR of View constants and module constants - let's keep to one or the other, perhaps a standard of module constants, starting in this commit?

Reply:
where else is `View` used? In tests View.LEFT_WIDTH has to be used because it is in parametrize.